jaffle_shop_online/models/marketing/schema.yml

```
@@ -1,298 +1,22 @@
-        data_type: varchar
-        description: "Source where the ad was displayed. For example, Google, Facebook, Twitter, Newsletter, etc."
-
-      - name: utm_medium
-        data_type: varchar
-        description: "The advertising or marketing medium. The medium can be social media, banners, CPC, third-party blogs, etc."
-
-      - name: utm_campain
-        data_type: varchar
-        description: "The name of the advertising campaign"
\ No newline at end of file
+      - name: return_on_advertising_spend
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)